Common/src/interpolation_structure.cpp

```diff
@@ -1028,61 +1106,89 @@ void CMirror::Set_TransferCoeff(CConfig **config) {
 

```

Comment:
Lots of changes in these routines, have CMirror and CIsoparametric been tested or only CNearestNeighbor has been validated?